certified-connectors/bitskout/apiDefinition.swagger.json

```diff
@@ -163,7 +163,737 @@
+							"file_url": {
+								"type": "string",
+								"description": "file_url"
+							}
+						}
+					}
+				}],
+				"x-ms-visibility": "important"
+			}
+		},
+		"/actions/cv": {
+			"post": {
+				"responses": {
+					"100": {
+						"description": "default",
+						"schema": {
+							"type": "object",
+							"properties": {
+								"outputs": {
+									"type": "object",
+									"properties": {
+										"EDUCATION": {
+											"type": "string",
+											"description": "EDUCATION"
+										},
+										"EMAILS": {
+											"type": "string",
+											"description": "EMAILS"
+										},
+										"EXPERIENCE": {
+											"type": "string",
+											"description": "EXPERIENCE"
+										},
+										"JOB_TITLE": {
+											"type": "string",
+											"description": "JOB_TITLE"
+										},
+										"LINKEDIN": {
+											"type": "string",
+											"description": "LINKEDIN"
+										},
+										"LOCATION": {
+											"type": "string",
+											"description": "LOCATION"
+										},
+										"NAME": {
+											"type": "string",
+											"description": "NAME"
+										},
+										"PHONE_NUMBERS": {
+											"type": "string",
+											"description": "PHONE_NUMBERS"
+										},
+										"RawJSON": {
+											"type": "string",
+											"description": "RawJSON"
+										},
+										"SKILLS": {
+											"type": "string",
+											"description": "SKILLS"
+										},
+										"TOTAL_YEARS_EXPERIENCE": {
+											"type": "string",
+											"description": "TOTAL_YEARS_EXPERIENCE"
+										}
+									},
+									"description": "outputs"
+								}
+							}
+						}
+					}
+				},
+				"summary": "Extract Data from CV (Resume)",
+				"description": "Extract data from CV (Resume) using predefined Bitskout plugin",
+				"operationId": "Extract_Data_From_CV",
+				"parameters": [{
+					"name": "body",
+					"in": "body",
+					"required": false,
+					"schema": {
+						"type": "object",
+						"properties": {
+							"file_url": {
+								"type": "string",
+								"description": "file_url"
+							}
+						}
+					}
+				}],
+				"x-ms-visibility": "important"
+			}
+		},
+		"/actions/doctype_{doctype}": {
+			"post": {
+				"responses": {
+					"100": {
+						"description": "default",
+						"schema": {
+							"type": "object",
+							"properties": {
+								"outputs": {
+									"type": "object",
+									"properties": {
+										"Document Type": {
+											"type": "string",
+											"description": "Document Type"
+										},
+										"RawJSON": {
+											"type": "string",
+											"description": "RawJSON"
+										}
+									},
+									"description": "outputs"
+								}
+							}
+						}
+					}
+				},
+				"summary": "Detect Document Type",
+				"description": "Detect the type of the provisioned document",
+				"operationId": "Detect_Document_Type",
+				"parameters": [{
+						"name": "doctype",
```

Comment:
@saferoombiz The 'x-ms-summary' property and the 'description' property is required. Please add both to the parameter.